### PR TITLE
add allowQueryParamWithNoEqual metatype prop

### PR DESCRIPTION
**Pull request by wtlucy**
For #12604

### CODE REVIEW

File: dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -176,4 +176,7 @@ stopAppStartUponListenerException.name=Stop application for unhandled listener e
...
 
 redirectToRelativeUrl.name=Send redirect response to a relative URL location
-redirectToRelativeUrl.desc=Send redirect response to a relative URL location without processing it. Set this property to true to send redirect response without converting the URL to an absolute location. 
\ No newline at end of file
+redirectToRelativeUrl.desc=Send redirect response to a relative URL location without processing it. Set this property to true to send redirect response without converting the URL to an absolute location.
+
+allowQueryParamWithNoEqual.name=Allow query parameters with no value
+allowQueryParamWithNoEqual.desc=If the query parameter in a URL contains only the string name instead of the expected name=value format, the server by default returns the value of request.getParameter(name) query as null.  When this property is set to true, if the query parameter in a URL contains only the string name, the server returns an empty string as the value for the request.getParameter(name) query.
```
Comment (jantley-ibm) [inline dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties:182]: Suggested rewrite:

allowQueryParamWithNoEqual.desc=When this property is set to true, if the query parameter in a URL contains only the string name, the server returns an empty string as the value for the request.getParameter(name) query. When this property is set to false, the server returns null as the value for the request.getParameter(name) query. The default value is false.

Comment (wtlucy) [reply dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties]: Thanks for the suggestion @jantley-ibm - I've made this change.

**Comment by wtlucy** [comment]
#build

**Comment by wtlucy** [comment]
#run-libby-bot

**Review by pmd1nh — APPROVED** [review]

**Comment by wtlucy** [comment]
All of the test failures in the previous build match existing failures - so I'll merge this.

**Pull request merged**